lib/ansible/modules/utilities/logic/_include.py

```diff
@@ -20,8 +20,9 @@
 module: include
 short_description: Include a play or task list
 deprecated:
-  The include action was too confusing, dealing with both plays and tasks, being both dynamic and static. This module
-  will be removed in version 2.8. As alternatives use M(include_tasks), M(import_playbook), M(import_tasks).
+  removed_in: "2.8"
+  why: The include action was too confusing, dealing with both plays and tasks, being both dynamic and static. This module will be removed in version 2.8.
+  alternative: Use M(include_tasks), M(import_playbook), M(import_tasks).
```

Comment:
Should this list of alternatives be expanded?

Reply:
Jimi said no